protos/plugin/codegen.proto

```diff
@@ -99,6 +100,12 @@ message GoCode
   string output_files_suffix = 18;
 }
 
+message WASM
+{
+  string path = 1;
+  string out = 2;
+}
```

Comment:
This is a temporary addition to make it easy to test out plugin support. I haven't figured out what the plugin config should look like yet.